web/src/tippyjs.ts

```diff
@@ -426,6 +426,24 @@ export function initialize(): void {
         },
     });
 
+    delegate("body", {
+        target: ".pill-container .pill",
+        // Disable trigger on focus, to avoid displaying on-click.
+        trigger: "mouseenter",
+        delay: LONG_HOVER_DELAY,
+        appendTo: () => document.body,
+        popperOptions: {
+            modifiers: [
+                {
+                    name: "flip",
+                    options: {
+                        fallbackPlacements: "bottom",
```

Comment:
This code will not be needed if you add `tippy-zulip-delayed-tooltip` class to html element.